cfme/common/vm.py

```diff
@@ -145,9 +145,9 @@ def factory(cls, vm_name, provider, template_name=None, template=False):
     TO_OPEN_EDIT = None  # Name of the item in Configuration that puts you in the form
     QUADICON_TYPE = "vm"
     # Titles of the delete buttons in configuration
-    REMOVE_SELECTED = {version.LOWEST: 'Remove selected items from the VMDB',
+    REMOVE_SELECTED = {'5.6.2.2': 'Remove selected items from the VMDB',
```

Comment:
I think that's in alignment with the read-the-docs style guide, to move first item in the dictionary to a newline so the keys line up.